# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dbus/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./mono/DBusType/Array.cs.orig	2005-03-19 18:03:55.342314760 -0400
+++ ./mono/DBusType/Array.cs	2005-03-19 18:04:19.056709624 -0400
@@ -59,7 +59,7 @@
       IntPtr arrayIter = Marshal.AllocCoTaskMem (Arguments.DBusMessageIterSize);
 
       if (!dbus_message_iter_open_container (iter,
-					     (int) this.Code,
+					     (int) Array.Code,
 					     Arguments.GetCodeAsString (elementType),
 					     arrayIter)) {
 	throw new ApplicationException("Failed to append array argument: " + val);
